<launch>

  <arg name="sim" default="false" />


  <!-- move_group -->
  <include file="$(find pr2_moveit_config)/launch/move_group.launch" if="$(arg sim)"/>
  <include file="$(find pr2_ft_moveit_config)/launch/move_group.launch" unless="$(arg sim)"/>

  <!-- load position dictionaries -->
  <rosparam command="load" file="$(find amazon_challenge_motion)/config/left_arm_joint_pos_dict.yaml" />
  <rosparam command="load" file="$(find amazon_challenge_motion)/config/right_arm_joint_pos_dict.yaml" />
  <rosparam command="load" file="$(find amazon_challenge_motion)/config/torso_joint_pos_dict.yaml" />
  <rosparam command="load" file="$(find amazon_challenge_motion)/config/base_pos_dict.yaml" />
  <rosparam command="load" file="$(find amazon_challenge_motion)/config/head_joint_pos_dict.yaml" />
      

  <!-- shelf collision model -->
  <node name="shelf_collision_planning_scene" pkg="amazon_challenge_motion" type="shelf_collision_planning_scene" output="screen" >
    <param name="base_frame_id" value="/base_laser_link"/>
    <param name="shelf_frame_id" value="/shelf_frame"/>
    <param name="translation_tolerance" value="0.003"/>
    <param name="rotation_tolerance" value="0.016"/>
  </node>

  <!-- tool collision model -->

  <node name="tool_collision_planning_scene" pkg="amazon_challenge_motion" type="tool_collision_planning_scene" output="screen"/>

</launch>